src/README.md

```diff
@@ -116,7 +116,7 @@ If we get can an average rating over 4/5.
 
 > Alternatives?
 
-Try [SlackEdit](https://stackedit.io/) if you prefer independent app!
+Try [SlackEdit](https://stackedit.io/) if you prefer an standalone app!
```

Comment:
```suggestion
Try [SlackEdit](https://stackedit.io/) if you prefer an independent app!
```
independent is actually different from standalone. independent means that it's a fantastic tool made by a tiny team. not made by the BigTechs, who had a bad reputation controlling our internet. standalone makes people feel that they're equivlent competitors who have the same budget. "Independent" encourage people to use their empathy to try it, while "standalone" discourages people to try it.